packages/neo-one-smart-contract-test/src/createNode.ts

```diff
@@ -46,7 +50,7 @@ export const createNode = async (omitCleanup = false) => {
           },
         },
       }),
-      leveldown: MemDown(),
+      leveldown: useLeveldown ? leveldown : MemDown(),
```

Comment:
should probably construct it here, IIRC calling `LevelDown` initializes the directory.